pandas-stubs/core/frame.pyi

```diff
@@ -188,6 +193,8 @@ class _LocIndexerFrame(_LocIndexer):
         | None,
     ) -> Series: ...
     @overload
+    def __getitem__(self, idx: tuple[ScalarT, slice]) -> Series | DataFrame: ...
```

Comment:
I think this will work with `Scalar` instead of `ScalarT`.  Can you try that?

Reply:
done sir